packages/dialtone-vue2/recipes/leftbar/general_row/general_row_constants.js

```diff
@@ -2,8 +2,8 @@ export const LEFTBAR_GENERAL_ROW_TYPES = {
   INBOX: 'inbox',
   CONTACTS: 'contacts',
   CHANNELS: 'channels',
-  HOME: 'home',
```

Comment:
huh, since we are removing `HOME` I found just one usage of it that we should update cos it will be deprecated.

`ubervoice/static/js/single/components/leftbar/leftbar_links.vue`

Reply:
Correct, I am updating that with another PR once this is merged. Thanks!